examples/ping-ring/main.go

```diff
@@ -95,12 +95,19 @@ func main() {
 
 	logger := log.StandardLogger()
 
+	rp, err := ringpop.New("ping-app",
+		ringpop.Channel(ch),
```

Comment:
`ringpop.Channel(...)`, `ringpop.Identity(...)` doesn't read as well as it could. I feel like these options should be qualified more clearly w/ config-related terms, for example:

```
ringpop.options.Identity(*hostport)
```

or 

```
ringpop.config.Identity(*hostport)
```

or

```
ringpop.IdentityOption(*hostport)
```

What do you think?